infrastructure/team-dev/gcp/au/routing/load-balancer.tf

```diff
@@ -0,0 +1,62 @@
+data "google_compute_zones" "this_env" {}
```

Comment:
All of this can be replicated more or less with the following in-cluster:
```yaml
kind: Gateway
apiVersion: gateway.networking.k8s.io/v1beta1
metadata:
  name: external-http
  annotations:
    networking.gke.io/certmap: store-example-com-map
spec:
  gatewayClassName: gke-l7-global-external-managed
  listeners:
  - name: https
    protocol: HTTPS
    port: 443
```

...where `store-example-com-map` is a reference to the cert generated in terraform in certificate-manager.

If you think the abstractions have come far enough @MattSurabian maybe it's time to fully go to the "dark side": on this.

Reply:
The world is going there. I still hate it for a few reasons but who am I to blow against the wind? Better queue up the https://github.com/kubernetes-sigs/external-dns deployments to go along with it